src/control/server/mgmt_pool.go

```diff
@@ -1,143 +1,175 @@
-//
-// (C) Copyright 2020 Intel Corporation.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-//
-// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
-// The Government's rights to use, modify, reproduce, release, perform, display,
-// or disclose this software are subject to the terms of the Apache License as
-// provided in Contract No. 8F-30005.
-// Any reproduction of computer software, computer software documentation, or
-// portions thereof marked with this legend must also reproduce the markings.
-//
-
-package server
-
-import (
-	"strings"
-
-	"github.com/golang/protobuf/proto"
-	"github.com/pkg/errors"
-	"golang.org/x/net/context"
-
-	mgmtpb "github.com/daos-stack/daos/src/control/common/proto/mgmt"
-	"github.com/daos-stack/daos/src/control/drpc"
-	"github.com/daos-stack/daos/src/control/server/ioserver"
-)
-
-// PoolCreate implements the method defined for the Management Service.
-//
-// Validate minimum SCM/NVMe pool size per VOS target, pool size request params
-// are per-ioserver so need to be larger than (minimum_target_allocation *
-// target_count).
-func (svc *mgmtSvc) PoolCreate(ctx context.Context, req *mgmtpb.PoolCreateReq) (*mgmtpb.PoolCreateResp, error) {
-	if req == nil {
-		return nil, errors.New("nil request")
-	}
+	//
+	// (C) Copyright 2020 Intel Corporation.
+	//
+	// Licensed under the Apache License, Version 2.0 (the "License");
+	// you may not use this file except in compliance with the License.
+	// You may obtain a copy of the License at
+	//
+	//    http://www.apache.org/licenses/LICENSE-2.0
+	//
+	// Unless required by applicable law or agreed to in writing, software
+	// distributed under the License is distributed on an "AS IS" BASIS,
+	// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	// See the License for the specific language governing permissions and
+	// limitations under the License.
+	//
+	// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+	// The Government's rights to use, modify, reproduce, release, perform, display,
+	// or disclose this software are subject to the terms of the Apache License as
+	// provided in Contract No. 8F-30005.
+	// Any reproduction of computer software, computer software documentation, or
+	// portions thereof marked with this legend must also reproduce the markings.
+	//
```

Comment:
What's with all of the whitespace changes?

Reply:
Hmm weird, not sure what caused that, I'll go back through and make sure I get rid of any unnecessary white space changes.